src/utils/loggerUtils.ts

```diff
@@ -0,0 +1,15 @@
+import { IZoweLogger, getZoweDir } from "@zowe/zowe-explorer-api";
+import { join as joinPaths } from "path";
+
+export class LoggerUtil {
```

Comment:
Can we make the class name consistent with the file name?
(personal opinion) I prefer the plural form 😋

Reply:
Yep! Should have been `LoggerUtils` for consistency, thanks for pointing that out 🙂 fixed 👍